src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -513,56 +401,9 @@ private static void DumpCmd(string prefix, int result, ref Guid pguidCmdGroup, u
 
         #region Caret and Cursor
 
-        private ITextCaret Caret => _textView.Caret;
-
         private void CaretPositionChanged(object sender, CaretPositionChangedEventArgs e)
```

Comment:
Is this event handler needed on the outer (non-UI thread) object? Events should only occur on the UI thread.